README.md

```diff
@@ -4,49 +4,36 @@
 
 > Meow. ~[Cat](https://en.wikipedia.org/wiki/Cat)
 
- 
-
-## Learning Objectives - The student should be able to..
+## Learning Objectives
 
 * Complete this lab.
 
+## Directions
+
+Cats: Our furry friends that make our lives better and are the reason the Internet exists. They're cute, they're lovable, and sometimes they even come in boxes.
+
+![Cat-in-the-Box](https://s3.amazonaws.com/learn-verified/cat-box.gif)
 
-## What the student can do at this point 
+In this lab, you're going to create an iOS app that will list a bunch of kitty breeds. When you're done, it will look like this:
 
-* Create variables and constants
-* Is familiar with type annotations, type inference and string interpolation.
-* Can create functions with return types.
-* Is familiar with the String, Int, Double, and Bool type.
-* Can perform arithmetic operations on Int and Double.
-* Understands if and else clause statements.
-* Can create and use Arrays.
-* Can iterate over an Array using a for-in loop.
-* Can iterate over an Array calling enumerate().
-* Work with the following methods on arrays:
-	* append()
-	* insert(_:atIndex:)
-	* removeAtIndex()
-	* subscript syntax with arrays
-	* count
-	* isEmpty
-* Can work with dictionaries 
-* Can iterate over a dictionary
-* Knows how to work with optionals
-* Classes
-* Structs
-* Properties
+![Cats!](https://s3.amazonaws.com/learn-verified/cats-ui.png)
 
-## Outline / Notes
+This app is backed by a table view, and also includes a navigation controller for managing the nav bar at the top of the screen.
 
-* The instructions should be pretty simple. Not too explicit.
-* Have them create a new Xcode project. I feel like we have to include this gif somehow.
+This lab shouldn't be too difficult, so you're not being given a lot of guidance in this one. However, if you thoroughly read through the previous lessons, you should have no problem. Feel free to refer to the previous two lessons on table views if you get stuck.
 
-![Cats](https://media.giphy.com/media/1ecRID74uAe5i/giphy.gif)
+Here are the tasks you must do in order to complete this app:
 
-* Given the list of cats (which you need to create), display them in  `UITableViewController` embedded in a `UINavigationController` which has the title set to Cats.  
+1. In `Main.storyboard`, drag a new **Table View Controller** from the object library to the storyboard.
+2. Don't forget to set `Main.storyboard` as the _initial view controller!_
```

Comment:
I will fix this, making note.
Line 28: Don't forget to set the **Table View Controller** as the _initial view controller!_